docs/modules/ROOT/pages/how-tos/update_maintenance.adoc

```diff
@@ -4,8 +4,20 @@
 +
 [source,console]
 ----
-kubectl --as cluster-admin get clusterversion version -o json | jq '.status.availableUpdates[] | {image: .image, version: .version}'
+oc adm upgrade --as cluster-admin
+
+Updates:
+
+VERSION IMAGE
+4.5.19  quay.io/openshift-release-dev/ocp-release@sha256:bae5510f19324d8e9c313aaba767e93c3a311902f5358fe2569e380544d9113e
+4.5.20  quay.io/openshift-release-dev/ocp-release@sha256:78b878986d2d0af6037d637aa63e7b6f80fc8f17d0f0d5b077ac6aca83f792a0
+4.5.24  quay.io/openshift-release-dev/ocp-release@sha256:f3ce0aeebb116bbc7d8982cc347ffc68151c92598dfb0cc45aaf3ce03bb09d11
 ----
+
+[NOTE]
+--
+If you don't get the newest available version, this might be intended. Red Hat does release new updates to specific cluster, when they do have no known issues. So on a stable channel you need some patience!
```

Comment:
```suggestion
If you don't get the newest available version, this might be intended.
Red Hat does release new updates to specific cluster, when they do have no known issues.
So on a stable channel you need some patience!
```